### PR TITLE
[monitoring] Add automated testing proxy (atproxy)

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
Hosting additional endpoints for automated testing from within a USS's primary systems can be challenging.  This new tool (atproxy) attempts to reduce this difficulty by providing a webserver implementing USS endpoints required by uss_qualifier for RID, providing implementation via incoming calls from elsewhere.  See [README.md in this PR](https://github.com/BenjaminPelletier/dss/blob/feature/atproxy/monitoring/atproxy/README.md) for more information.

### CODE REVIEW

File: monitoring/atproxy/handling.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+def fulfill_query(req: ImplicitDict, logger: logging.Logger) -> Tuple[str, int]:
+    """Fulfill an incoming automated testing query.
+
+    :param req: Request descriptor from requests.py.
+    :param logger: Flask logger.
+    :return: Flask endpoint handler result (content, HTTP code).
+    """
+    t_start = datetime.utcnow()
+    query = Query(type=req.request_type_name(), request=req)
+    timeout = timedelta(seconds=59)
```
Comment (barroco) [inline monitoring/atproxy/handling.py:22]: Manual testing showed that Gunicorn default timeout of 30 seconds interferes with this.
[`--timeout [SEC]` may be added to start.sh to align timeouts.](https://docs.gunicorn.org/en/stable/settings.html#timeout)

**Review by barroco — APPROVED** [review]
@BenjaminPelletier this looks great thanks!

File: monitoring/atproxy/__init__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import logging
+_logger = logging.getLogger('atproxy.context')
+_logger.setLevel(logging.DEBUG)
+_logger.info('Proto message')
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply monitoring/atproxy/__init__.py:4]: Yes, thanks -- removed.

File: monitoring/atproxy/atproxy.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+									"key": "issuer",
+									"value": "localhost"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get RID injection token from dummy OAuth Copy",
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply monitoring/atproxy/atproxy.postman_collection.json:62]: Done.

File: monitoring/atproxy/handling.py (hunk 1 above)
Comment (BenjaminPelletier) [reply monitoring/atproxy/handling.py:22]: Thanks; added `--timeout 60` to start.sh

File: monitoring/atproxy/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+`atproxy` ("automated testing proxy") contains a proxy server to host automated
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply monitoring/atproxy/README.md:1]: Good point; it is now listed as a diagnostic tool in the monitoring README.

**Pull request merged**